app/controllers/authors_controller.rb

```diff
@@ -228,6 +228,7 @@ def extension
   end
 
   def update
+    @author = @display_author
```

Comment:
So..`@author` is set here: https://github.com/standardnotes/listed/blob/develop/app/controllers/application_controller.rb#L16

`@author` refers to an authenticated instance of an author. It is only set if the secret key is correct. `@display_author` is only used when we are referencing the author for public, non-authenticated purposes. So by setting `@author = @display_author` here, you would be essentially bypassing authentication and allowing updates on an author by just passing their ID (since `@display_author` is set in the `before_action` filter in this controller, based on simple params).

Reply:
Solved this by sending secret as a parameter in the request, for some reason I hadn't realised this was the way to fix it. Thank you for your help!